@xen-orchestra/lite/src/stories/web-core/ui/tag/ui-tags-list.story.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      iconProp(),
+      prop('accent')
+        .type('TagAccent')
+        .required()
+        .preset('info')
+        .enum('info', 'neutral', 'success', 'warning', 'danger', 'muted')
+        .widget(),
+      prop('variant').type('TagVariant').required().preset('primary').enum('primary', 'secondary').widget(),
+      slot(),
+      setting('slot').preset('Label').widget(),
+    ]"
+  >
+    <UiTagsList>
+      <UiTag v-bind="properties">{{ settings.slot }}</UiTag>
+      <UiTag v-bind="properties">{{ settings.slot }}</UiTag>
+      <UiTag v-bind="properties">{{ settings.slot }}</UiTag>
+      <UiTag v-bind="properties">{{ settings.slot }}</UiTag>
```

Comment:
To handle the removal of `properties':
```suggestion
      <UiTag accent="info" variant="primary">{{ settings.slot }}</UiTag>
      <UiTag accent="success" variant="primary">{{ settings.slot }}</UiTag>
      <UiTag accent="warning" variant="secondary">{{ settings.slot }}</UiTag>
      <UiTag accent="danger" variant="secondary">{{ settings.slot }}</UiTag>
```